src/bootstrap/config.rs

```diff
@@ -1378,6 +1394,19 @@ fn set<T>(field: &mut T, val: Option<T>) {
     }
 }
 
+#[derive(Deserialize)]
+struct Stage0 {
+    config: Stage0Config,
```

Comment:
My rustfmt commit also introduced a stage0 struct - do you mind waiting for that to merge sowe don't have duplicates? I'll try and get that done today.

Reply:
Sure! I'll wait and then rebase :sligh